src/backends/tensorflow/base_backend.cc

```diff
@@ -449,6 +449,7 @@ ReadFixedSizedOutputTensor(
       void* content = nullptr;
       Status status = payload.response_provider_->AllocateOutputBuffer(
           output_name, &content, expected_byte_size, shape);
+      LOG_VERBOSE(1) << "Allocated Output Buffer";
```

Comment:
Either add more information to this log, like tensor name, output buffer address and size.
Or remove this line

Reply:
Added for debugging. Removing.